eg/basic-example.js

```diff
@@ -0,0 +1,67 @@
+// - Basic Example
+// A bare minimum example of two subscribers
+// handling some messages.
+
+'use strict'
```

Comment:
This repo is not using 'use strict' everywhere, however I am ok to add it. In case, place it everywhere.